CIFAR10/train/diux_cifar_g3.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "v1.0",
+    "author": "DIUX",
+    "authorEmail": "ybulatov.ctr@diux.mil",
+    "framework": "tensorflow 1.5, tensorpack 0.8.1",
+    "codeURL": "https://github.com/ppwwyyxx/tensorpack/blob/master/examples/ResNet/cifar10-resnet.py",
+    "model": "ResNet50",
+    "hardware": "g3.4xlarge",
+    "timestamp": "2018-1-31",
+    "misc": "ami-9ba7c4e1 (Amazon Deep Learning AMI version 2, Ubuntu, conda based) "
```

Comment:
You're missing a comma at the end of this line. Also, "misc" should be an object. Please include a key for this value. Here is an [example](https://github.com/stanford-futuredata/dawn-bench-entries/blob/844790a9e4cae17eb545b8fd67b9150a55219a1c/CIFAR10/train/dawn_resnet164_b_1k80-gc_pytorch.json#L11). Also, can you include your learning rate schedule and optimizer. From your, codeURL it looks like it should be:
```
"optimizer": "SGD with Momentum",
"momentum": 0.9,
"schedule": [
    {"learning_rate": 0.1, "epochs": 82},
    {"learning_rate": 0.01, "epochs": 41},
    {"learning_rate": 0.001, "epochs": 177},
    {"learning_rate": 0.0002, "epochs": 100}
],
# Please check that this is valid JSON before committing
```